src/structure/module.rs

```diff
@@ -59,3 +64,115 @@ pub trait VectorSpace<S: Field>
     : VectorSpaceApprox<S>
     + Module<S>
 {}
+
+
+/// A normed approximate vector space.
+pub trait NormedSpaceApprox<S: FieldApprox>
+    : VectorSpaceApprox<S> {
```

Comment:
I'm thinking that we should eventually make these type parameters associated types - ie. `VectorSpace::Scalar` - see [cgmath's trait](http://brendanzab.github.io/cgmath/cgmath/trait.VectorSpace.html#associatedtype.Scalar).